packages/chain-helpers/src/errorhandling/ErrorHandler.spec.ts

```diff
@@ -24,20 +23,17 @@ describe('ErrorHandler', () => {
         },
       },
     }
-    const submittableResult: ISubmittableResult = {
-      // @ts-ignore
-      events: [evtRecord],
-    }
+    const submittableResult = {
+      events: ([evtRecord] as unknown) as EventRecord[],
```

Comment:
a lot of type assertions here. Is there a way to make it work without enforcing the types?

Reply:
`@ts-ignore` as we had it before :D

Reply:
this is testing, right, so we're using a mocked EventRecord. Could potentially construct the real thing using the TypeRegistry, but is it worth the trouble?

Reply:
oh yeah, sure. You are right. Not worth the trouble